Xamarin.Forms.DualScreen/Xamarin.Forms.DualScreen.csproj

```diff
@@ -13,15 +13,6 @@
     <XamarinAndroidSupportSkipVerifyVersions>true</XamarinAndroidSupportSkipVerifyVersions>
     <GenerateLibraryLayout Condition=" $(TargetFramework.StartsWith('uap10.0')) ">false</GenerateLibraryLayout>
   </PropertyGroup>
-  <PropertyGroup Condition="$(TargetPlatformVersion) == 'uap10.0.14393'">
-    <DefineConstants>$(DefineConstants);UWP_14393</DefineConstants>
-  </PropertyGroup>
-  <PropertyGroup Condition="$(TargetPlatformVersion) == 'uap10.0.16299'">
-    <DefineConstants>$(DefineConstants);UWP_18362</DefineConstants>
```

Comment:
I see `#if UWP_18362` referenced in the uwp code. I might be missing something, maybe at this stage it's intended to be excluded?